CaptainMumo/stackoverflow-lite
# #162571199 Add feature for searching for a question

#### What does this PR do?
Add feature to search for a specific question
#### Description of Task to be completed?
- Create a search bar on the home page
#### How should this be manually tested?
Clone this repo, open UI folder on a text/code editor then run index.html on a browser to see the search bar
#### Any background context you want to provide?
The search function will be seen working when the API function is implemented
#### What are the relevant pivotal tracker stories?
#162571199
#### Screenshots
![image](https://user-images.githubusercontent.com/39964147/49902208-5c1f0b80-fe74-11e8-830b-02eed6209729.png)
